src/subscription/apis.ts

```diff
@@ -0,0 +1,92 @@
+export const FreeApis = [
```

Comment:
@Chimexx we dont need to save all these, all we need to save is the apiId, the base url and the route we want to hit.

Reply:
Okay, I will refactor that